documentation/modules/deploying/con-fips-support.adoc

```diff
@@ -21,6 +21,9 @@ You can do that, for example, by deleting the user secret and waiting for the Us
 IMPORTANT: If you are using FIPS-enabled Kubernetes clusters, you may experience higher memory consumption compared to regular Kubernetes clusters. 
 To avoid any issues, we suggest increasing the memory request to at least 512Mi.
 
+//FIPS NIST validation
+include::../../modules/deploying/con-fips-validation.adoc[leveloffset=+1]
```

Comment:
Not directly related to this change. But would it make more sense to have it in the following order:
1. The intro / abstract
2. The `IMPORTANT` warning about the resources
3. The new section about the validation
4. The section about minimum password length

I think that would better correspond the importance. (technically, the memory things could be after the FIPS validation section. But it would look strange as the warning does not have its title)